lib/index.js

```diff
@@ -5,10 +5,7 @@ var Resource = require('./Resource'),
     Controllers = require('./Controllers'),
     Errors = require('./Errors'),
     inflection = require('inflection'),
-    _ = require('lodash');
-
-// make deep defaults the default
-_.defaults = require('merge-defaults');
+    mergeDefaults = require('merge-defaults');
```

Comment:
Did you actually run into an issue with this? The thing is that we aren't actually sharing lodash with sequelize, they use an internal copy of it and export it via Utils._, but we depend on lodash directly and therefore there should be no overlap